documentation/release_notes.rst

```diff
@@ -8,6 +8,92 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.7.0
+===============
+News
+------------
+
+New Features
+------------
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``, ``is_partitioned``, and ``lexicographical_compare`` algorithms with device policies. 
+- Improved performance of ``exclusive_scan``, ``inclusive_scan``, ``transform_exclusive_scan``, ``transform_inclusive_scan``, ``copy_if``, ``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if'', ``partition``, ``partition_copy``, ``stable_partition``, ``unique``, and ``unique_copy`` algorithms on GPU devices.
+- Improved performance of ``adjacent_find``, ``all_of``, ``any_of``, ``equal``, ``find``, ``find_if``, ``find_end``, ``find_first_of``, ``find_if_not``, ``includes``, ``is_heap``, ``is_heap_until``, ``is_sorted``, ``is_sorted_until``, ``mismatch``, ``none_of``, ``search``, and ``search_n`` algorithms on GPU devices.
```

Comment:
```suggestion
- Improved performance on GPU devices of ``scan`` , ``copy_if``, ``remove'', ``partition``, ``unique``, ``find``, ``is_heap``, ``is_sorted``, and ``search`` families of algorithms, as well as ``[all/any/none]_of``, ``equal``, ``includes``, and ``mismatch`` APIs.
```

I think it has been our convention to list all algorithms out, but perhaps we can shorten it like this or similar to make it more readable.

Reply:
I hesitate to use the algorithms families because I don't know how widely known they are. Do others have an opinion?  I agree the shorter version is more readable.

Reply:
Let me put the full list together as @SergeyKopienko suggested and let's see how that looks compared to the shorter suggestion made here.

Reply:
I agree that it makes it less precise.  I've grouped the "families" by name rather than by underlying implementation details (copy_if separate from remove even though they are the same under the hood).  

We could instead list them out similarly to `[all/any/none]_of`:  Lets see what that would look like: 

```
Improved performance on GPU devices of `[transform_][inclusive/exclusive_]scan` , ``copy_if``, ``remove[_copy][_if]``, ``[stable_]partition[_copy]``, ``unique[_copy]``, ``find[_if/_end/_first_of/_if_not]``, ``is_heap[_until]``, ``is_sorted[_until]``, and ``search[_n]``, ``[all/any/none]_of``, ``equal``, ``includes``, and ``mismatch`` APIs.
```

We should probably sort, but first we can figure out our strategy and whether or not we want to group this with the above list as well.

Reply:
The changes are made. @danhoeflinger and @SergeyKopienko would you please take a look at the line and weigh in between that and Dan's suggestion here?

Reply:
Its going to be a long and difficult to read list either way I think.  Perhaps its best to just include them all explicitly because then it is at least searchable, which may as important than readable.

Reply:
Just a note:  
It is technically till true that for the scan-like algorithms, we did improve performance even on the CPU with a device policy, just not as dramatically as on the GPU (this is from the switch of the temp storage and return type).  
I think the grouping under improvements for device policies is fine for scan-like algorithms.

Reply:
I think that Improved performance list looks ok now.

Reply:
same, resolving this.